cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -2458,6 +2458,37 @@
                       }
                     }
                   }
+                },
+                {
+                  "Fn::If": [
+                    "CreateHITSubstack",
```

Comment:
nice! can you please add this conditional also to the ec2 policies in the 2 blocks above?